activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -976,6 +976,9 @@ def prepend(*args)
       # Equivalent to +delete_all+. The difference is that returns +self+, instead
       # of an array with the deleted objects, so methods can be chained. See
       # +delete_all+ for more information.
+      # Note that because +delete_all+ removes records directly by
+      # running a SQL query into database, the +updated_at+ column of
```

Comment:
I guess you should write "by directly running an SQL query into the database", what do you think ?